configuration/cdap-site.xml

```diff
@@ -576,6 +598,13 @@
   </property>
 
   <property>
+    <name>data.tx.pruning.plugin.class</name>
+    <value>co.cask.data2.txprune.DefaultHBaseTransactionPruningPlugin</value>
+    <description>Class name for the default transaction pruning plugin</description>
+    <display-name>Transaction Client Pruning Plugin Class</display-name>
+  </property>
+
+  <property>
     <name>data.tx.snapshot.codecs</name>
```

Comment:
Remove this property, per [here](https://issues.cask.co/browse/CDAP-8393?focusedCommentId=28133&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-28133)